hbase-common/src/main/java/org/apache/hadoop/hbase/trace/HBaseSemanticAttributes.java

```diff
@@ -28,7 +28,9 @@
 */
 @InterfaceAudience.Private
 public final class HBaseSemanticAttributes {
+  public static final AttributeKey<String> DB_NAME = SemanticAttributes.DB_NAME;
   public static final AttributeKey<String> NAMESPACE_KEY = SemanticAttributes.DB_HBASE_NAMESPACE;
+  public static final AttributeKey<String> DB_OPERATION = SemanticAttributes.DB_OPERATION;
   public static final AttributeKey<String> TABLE_KEY = AttributeKey.stringKey("db.hbase.table");
```

Comment:
I guess we can drop the `_KEY` part here as none of these constants we import from `SemanticAttributes` use this naming convention.